db/migrate/20210115135535_add_global_chat.rb

```diff
@@ -0,0 +1,10 @@
+class AddGlobalChat < ActiveRecord::Migration[6.0]
+  def up
+    Room.create name: 'Global Chat', id: Room::GLOBAL_CHAT_ID
```

Comment:
Because this room exists in all instances of our software we should think of a way to use I18n

Reply:
That's a good point. However the name shown here is not used to display the name on the dashboard. That is already done using I18n.